completions/brctl

```diff
@@ -1,4 +1,12 @@
 # bash completion for brctl                                -*- shell-script -*-
+_comp_xfunc_brctl_interfaces()
+{
+    _comp_split COMPREPLY "$(${1:-brctl} show ${2:+"$2"} 2>/dev/null | _comp_awk \
+        '(NR == 1) { next }; (/^\t/) { print $1; next }; { print $4 }'))"
+
+    ((${#COMPREPLY[@]})) &&
+        _comp_split COMPREPLY "$(compgen -W '"${COMPREPLY[@]/%[[:punct:]]/}"' -- "$cur")"
```

Comment:
```suggestion
    _comp_compgen_split -- "$(${1:-brctl} show ${2:+"$2"} 2>/dev/null | _comp_awk \
        '(NR == 1) { next }; (/^\t/) { sub(/[[:punct:]]$/, "", $1); print $1; next }
        { sub(/[[:punct:]]$/, "", $4); print $4 }'))"
```

Reply:
Done. `sub(/[[:punct:]]$/, "", $1);` isn't actually necessary.

Reply:
Thanks!

> Done. `sub(/[[:punct:]]$/, "", $1);` isn't actually necessary.

Great, my suggestion was just an equivalent transform, without considering the output format of `brctl show`. So we can skip that if it is unnecessary.